api/src/main/resources/liquibase.xml

```diff
@@ -1020,6 +1020,32 @@ VALUES
             WHERE pp.program_id IN (5,6,7,8,9);
         </sql>
     </changeSet>
+
+	<changeSet id="ugandaemr-20240610-1655" author="slubwama">
+		<preConditions onFail="CONTINUE">
+			<sqlCheck expectedResult="1">
+				select count(*) from concept where uuid="7cac8397-53cd-4f00-a6fe-028e8d743f8e";
+			</sqlCheck>
+		</preConditions>
+		<comment>Change Image attachment to known concept metadata in UgandaEMR</comment>
+		<sql>
+			update concept_name set concept_name.uuid="552e757e-b072-4e68-a10d-b7db7efcd5ad", concept_name.concept_name_id=198844, concept_name.concept_id=186351  where concept_id in (select concept.concept_id from  concept where concept.uuid="7cac8397-53cd-4f00-a6fe-028e8d743f8e");
```

Comment:
`Caused by: liquibase.exception.DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`dev-plus`.`concept_name`, CONSTRAINT `concept_name_ibfk_1` FOREIGN KEY (`concept_id`) REFERENCES `concept` (`concept_id`)) [Failed SQL: (1452) update concept_name set concept_name.uuid="552e757e-b072-4e68-a10d-b7db7efcd5ad", concept_name.concept_name_id=198844, concept_name.concept_id=186351  where concept_id in (select concept.concept_id from  concept where concept.uuid="7cac8397-53cd-4f00-a6fe-028e8d743f8e")]
`

Reply:
@akileng56 reset concept dic then restart